### PR TITLE
Add back the internal database configuration docs

### PR DESCRIPTION
**Pull request by agrare**
At some point the internal database configuration appears to have been lost from the docs.

Move the external database instructions to the configuration-db include so that the instructions show up on all of the installation guides.

Below you can see we get "Configuring a database" with one "Configure the appliance to use an external PostgreSQL database" option.

image

### CODE REVIEW

**Comment by Fryguy** [comment]
I know this doc was messed with because of downstream concerns.  Do you know what PR removed it?

**Comment by kbrock** [comment]
@Fryguy can you take another look at this?

**Comment by agrare** [comment]
I think this is still a valid issue, marking `pinned`

**Comment by Fryguy** [comment]
Ok, it took me a bit to understand what happened, but the goal of that other PR was to extract the internal database into a separate file, so that we could exclude it downstream (but not exclude external database docs).  I actually like that this PR extracts the external db docs to a separate file as well, however, it should move into a different file than the internal db docs.  See https://github.com/ManageIQ/manageiq-documentation/pull/1689/files#r1308798623

File: _includes/configuration.md [hunk 1]
```diff
@@ -37,64 +37,10 @@ environment. Before using {{ site.data.product.title_short }}, configure the dat
...
-    (`vmdb_production`).
-
-8.  Enter the database username or leave blank for the default (`root`).
-
-9. Enter the chosen database user’s password.
-
-10. Confirm the configuration if prompted.
-
-{{ site.data.product.title_short }} will then configure the external database.
+{% include configuration-db.md %}
```
[the remark this answers is not in the document]
Comment (agrare) [reply _includes/configuration.md]: :+1: done

File: _includes/external-configuration-db.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+  - Because the `postgresql.conf` file controls the operation of all
+    databases managed by a single instance of PostgreSQL, do not mix
+    {{ site.data.product.title_short }} databases with other types of databases in a single
+    PostgreSQL instance.
+
+1.  Start the appliance and open a terminal console.
+
+2.  Enter the `appliance_console` command. The {{ site.data.product.title_short }} appliance
+    summary screen displays.
```
Comment (bdunne) [inline _includes/external-configuration-db.md:37]:
```suggestion
2.  Enter the `appliance_console` command. The summary screen will be displayed.
```

File: _includes/configuration.md [hunk 3]
```diff
@@ -39,64 +39,11 @@ environment. Before using {{ site.data.product.title_short }}, configure the dat
 options for it; {{ site.data.product.title_short }} provides the following two
 options for database configuration:
 
+  - Configure the appliance to use an internal PostgreSQL database
```
[the remark this answers is not in the document]
Comment (agrare) [reply _includes/configuration.md]: I moved things around a little bit, I moved this description block into `configuration-db.md` which then includes the two `-internal.md` and `-external.md` files.  I think that covers this case as well as simplifies the openstack quickstart guide (why do we only have that for openstack?)

**Comment by Fryguy** [comment]
Backported to `quinteros` in commit 6b4202a9e2e00ed3fe90b4474949f52d55b008cd.

```text
commit 6b4202a9e2e00ed3fe90b4474949f52d55b008cd
Author: Jason Frey <fryguy9@gmail.com>
Date:   Wed Aug 30 15:33:38 2023 -0400

    Merge pull request #1689 from agrare/add_back_internal_database_configuration
    
    Add back the internal database configuration docs
    
    (cherry picked from commit b5c5be4fbde2f2325e5b62a7780c00b5b470cae3)
```

**Pull request merged**